ui/app/templates/vault/cluster/recovery/snapshots/snapshot/manage.hbs

```diff
@@ -0,0 +1,6 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Recovery::Page::Snapshots::Snapshot::Manage />
```

Comment:
If you agree with my routing comment above, we'll want to remember to rename this file to `recovery/snapshots/snapshot.hbs` (It could also just be `snapshot.hbs` but I kind of like when the "base" route is named index

And the page component renamed so it's: `<Recovery::Page::Snapshots::Snapshot />`